internal/db/db_sqlite.go

```diff
@@ -187,6 +222,56 @@ func (s *Sqlite) DeleteFile(ctx context.Context, id string) error {
 	return nil
 }
 
+func (s *Sqlite) FindFiles(ctx context.Context, page int) ([]*snips.File, error) {
```

Comment:
nit: i'd rename this to be more specific

```suggestion
func (s *Sqlite) LatestPublicFiles(ctx context.Context, page int) ([]*snips.File, error) {
```